core/new-gui/src/app/dashboard/component/feature-container/user-project-section/project-page/ngbd-modal-add-project-file/ngbd-modal-add-project-file.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, Input, OnInit } from "@angular/core";
```

Comment:
The add project file, add project workflow, remove project file remove project workflow have too many logic duplicates. please consider make some utility calls

Reply:
Will unify into a single, multi-purpose modal in future PR